vmaas/reposcan/database/csaf_store.py

```diff
@@ -46,7 +48,125 @@ def _save_csaf_files(self, csaf_files: CsafFiles) -> list[int]:
         self.conn.commit()
         return db_ids
 
+    def _get_existing_cve_products(self, cve) -> Set[tuple]:
+        existing_products = set()
+        try:
+            cur = self.conn.cursor()
+            cur.execute("SELECT csaf_product_id FROM csaf_cves WHERE cve = %s", (cve,))
+            ids = [row[0] for row in cur.fetchall()]
+            if ids:
+                cur.execute("SELECT * from csaf_products where id in %s", (tuple(ids),))
+                for row in cur.fetchall():
+                    existing_products.add(row)
+
+        except Exception as exc:
+            CSAF_FAILED_IMPORT.inc()
+            self.logger.exception("Failure while fetching CSAF products from the DB: %s", exc)
+        finally:
+            cur.close()
+
+        return existing_products
+
+    def _get_product_rows(self, cve, import_products):
+        cve_db_products = self._get_existing_cve_products(cve)
+
+        to_insert_products = import_products - cve_db_products
```

Comment:
This doesn't work correctly because format of these tuples is not same

`import_products` tuples are in format:
```
('cpe:/o:redhat:enterprise_linux:9', 'kernel-rt', 4, None)
```

but `cve_db_products` tuples are in format:
```
(141, 'cpe:/o:redhat:enterprise_linux:9', 'kernel-rt', None)
```